### PR TITLE
7.x lcp dt fix

### PR DESCRIPTION
**Pull request by MorganDawe**

### CODE REVIEW

File: platform_main_feature.views_default.inc [hunk 1]
```diff
@@ -270,15 +270,15 @@ function platform_main_feature_views_default_views() {
...
-  /* Field: Islandora Solr: lcp_mods_facet_date_mdt */
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['id'] = 'lcp_mods_facet_date_mdt';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['table'] = 'islandora_solr';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['field'] = 'lcp_mods_facet_date_mdt';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['label'] = '';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['element_label_colon'] = FALSE;
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['link_to_object'] = 0;
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['date_format'] = 'custom';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['custom_date_format'] = 'Y-m-d';
+  /* Field: Islandora Solr: lcp_marc_facet_date_dt */
```
Comment (nigelgbanks) [inline platform_main_feature.views_default.inc]: Why the change to MARC, ticket seems to suggest the date come from MODS, I'm not sure all objects actually have MARC data, but all should have MODS (Mapped from MARC, QDC, CBS)?

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Guess I was wrong there is marc date being populated (though this object has no marc datastream):

screen shot 2016-11-25 at 12 17 40

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Added for LCP-49.

https://github.com/discoverygarden/lcp-basic-solr-config/commit/c9388c92d6368f9d7d8016218227efc4ffb61fba

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Though LCP-49 doesn't make mention of it? https://discoverygarden.atlassian.net/browse/LCP-49

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Still `lcp_mods_facet_date_mdt` is being populated, why not use it, as is indicated in the ticket, it would be a bit clearer I think. Still haven't managed to find out why this marc field was added though.

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Or did the `*_mdt` cause the problems with rendering?

**Review by nigelgbanks — COMMENTED** [review]
Just a question.

File: platform_main_feature.views_default.inc (hunk 1 above)
Comment (MorganDawe) [reply platform_main_feature.views_default.inc]: _mdt being a multi valued field prevented it from rendering properly.

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: @jordandukart since you added the marc field in discoverygarden/lcp-basic-solr-config@c9388c9 can you clarify why it's there? perhaps we should have a _dt field for the mods_facet as well, and remove the marc_facet fields? Or are they there for a reason?

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Marc field is fine to stay in the indexing just named a bit odd since it doesn't have the mods prefix, but no worries. 

Not 100% sure if we will always have `@encoding = 'marc'` in the mods. There may be cases in the data where `@encoding` does not equal 'marc'. Pluss this `marc` value is only using dateIssued where as the ticket calls for dateCreated?

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: The field first_date field which will take either date issued or date created might make the most sense (it's not a date field by a string field).

https://github.com/discoverygarden/lcp-basic-solr-config/blob/c9388c92d6368f9d7d8016218227efc4ffb61fba/islandora_transforms/lcp_MODS_to_solr.xslt#L170-L175

Probably needs some clarification from @desmodude750 as to what the client actually wants.

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: From the ticket:

>  Also make sure that only the date shows, not a dateTime value.

Hmm seems like first date would probably fit this best then since it's not been formatted to ISO.

Comment (nigelgbanks) [reply platform_main_feature.views_default.inc]: Lets do first date, and see what Stephen says come Monday

File: platform_main_feature.info [hunk 2]
```diff
@@ -263,3 +262,4 @@ features_exclude[menu_custom][menu-footer-menu] = menu-footer-menu
 features_exclude[menu_custom][menu-header-menu] = menu-header-menu
 features_exclude[menu_custom][main-menu] = main-menu
 features_exclude[menu_custom][menu-social-media-header] = menu-social-media-header
+features_exclude[menu_custom][user-menu] = user-menu
```
Comment (MorganDawe) [inline platform_main_feature.info:265]: Excluding this, as it is already included in the content type feature.

**Review by nigelgbanks — APPROVED** [review]
